pip install ddtrace==0.38.1 failed on Windows 10

`pip install ddtrace==0.38.1` failed on Windows 10 with at least python 3.8 . The last version that could be installed in the 0.36.1.

My python version is Windows 64 bits 3.8.3

```powershell
PS C:\Users\opiot> python --version
Python 3.8.3
PS C:\Users\opiot> python
Python 3.8.3 (tags/v3.8.3:6f8c832, May 13 2020, 22:37:02) [MSC v.1924 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> quit()
```

Creation of a clean new virtual env

```powershell
PS C:\Users\opiot> python -m venv venv/dd
PS C:\Users\opiot> .\venv\dd\Scripts\Activate.ps1
```



```powershell
(dd) PS C:\Users\opiot> pip freeze
(dd) PS C:\Users\opiot>
```



Upgrade pip to the last version

```powershell
(dd) PS C:\Users\opiot> python -m pip install --upgrade pip
Collecting pip
  Using cached https://files.pythonhosted.org/packages/43/84/23ed6a1796480a6f1a2d38f2802901d078266bda38388954d01d3f2e821d/pip-20.1.1-py2.py3-none-any.whl
Installing collected packages: pip
  Found existing installation: pip 19.2.3
    Uninstalling pip-19.2.3:
      Successfully uninstalled pip-19.2.3
Successfully installed pip-20.1.1
(dd) PS C:\Users\opiot> pip --version
pip 20.1.1 from c:\users\opiot\venv\dd\lib\site-packages\pip (python 3.8)
```

Install the dd-trace

```powershell
(dd) PS C:\Users\opiot> python -m pip install ddtrace==0.38.1
Collecting ddtrace==0.38.1
  Using cached ddtrace-0.38.1.tar.gz (887 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
    Preparing wheel metadata ... done
Collecting msgpack>=0.5.0
  Using cached msgpack-1.0.0-cp38-cp38-win_amd64.whl (73 kB)
Building wheels for collected packages: ddtrace
  Building wheel for ddtrace (PEP 517) ... error
  ERROR: Command errored out with exit status 1:
   command: 'C:\Users\opiot\venv\dd\Scripts\python.exe' 'C:\Users\opiot\venv\dd\lib\site-packages\pip\_vendor\pep517\_in_process.py' build_wheel 'C:\Users\opiot\AppData\Local\Temp\tmpoztvdfqw'
       cwd: C:\Users\opiot\AppData\Local\Temp\pip-install-t1aocqdk\ddtrace
  Complete output (604 lines):
  [1/4] Cythonizing ddtrace/internal/_rand.pyx
  [2/4] Cythonizing ddtrace/profiling/_build.pyx
  [3/4] Cythonizing ddtrace/profiling/collector/_traceback.pyx
  [4/4] Cythonizing ddtrace/profiling/collector/stack.pyx
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.8
  creating build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\api.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\compat.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\constants.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\context.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\encoding.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\filters.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\helpers.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\monkey.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\payload.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\pin.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\provider.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\sampler.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\span.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\tracer.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\util.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\_hooks.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\_worker.py -> build\lib.win-amd64-3.8\ddtrace
  copying ddtrace\__init__.py -> build\lib.win-amd64-3.8\ddtrace
  creating build\lib.win-amd64-3.8\ddtrace\bootstrap
  copying ddtrace\bootstrap\sitecustomize.py -> build\lib.win-amd64-3.8\ddtrace\bootstrap
  copying ddtrace\bootstrap\__init__.py -> build\lib.win-amd64-3.8\ddtrace\bootstrap
  creating build\lib.win-amd64-3.8\ddtrace\commands
  copying ddtrace\commands\ddtrace_run.py -> build\lib.win-amd64-3.8\ddtrace\commands
  copying ddtrace\commands\__init__.py -> build\lib.win-amd64-3.8\ddtrace\commands
  creating build\lib.win-amd64-3.8\ddtrace\contrib
  copying ddtrace\contrib\util.py -> build\lib.win-amd64-3.8\ddtrace\contrib
  copying ddtrace\contrib\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib
  creating build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\aws.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\cassandra.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\consul.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\db.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\elasticsearch.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\errors.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\http.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\kombu.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\memcached.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\mongo.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\net.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\priority.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\redis.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\sql.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\system.py -> build\lib.win-amd64-3.8\ddtrace\ext
  copying ddtrace\ext\__init__.py -> build\lib.win-amd64-3.8\ddtrace\ext
  creating build\lib.win-amd64-3.8\ddtrace\http
  copying ddtrace\http\headers.py -> build\lib.win-amd64-3.8\ddtrace\http
  copying ddtrace\http\__init__.py -> build\lib.win-amd64-3.8\ddtrace\http
  creating build\lib.win-amd64-3.8\ddtrace\internal
  copying ddtrace\internal\context_manager.py -> build\lib.win-amd64-3.8\ddtrace\internal
  copying ddtrace\internal\hostname.py -> build\lib.win-amd64-3.8\ddtrace\internal
  copying ddtrace\internal\import_hooks.py -> build\lib.win-amd64-3.8\ddtrace\internal
  copying ddtrace\internal\logger.py -> build\lib.win-amd64-3.8\ddtrace\internal
  copying ddtrace\internal\rate_limiter.py -> build\lib.win-amd64-3.8\ddtrace\internal
  copying ddtrace\internal\writer.py -> build\lib.win-amd64-3.8\ddtrace\internal
  copying ddtrace\internal\__init__.py -> build\lib.win-amd64-3.8\ddtrace\internal
  creating build\lib.win-amd64-3.8\ddtrace\opentracer
  copying ddtrace\opentracer\helpers.py -> build\lib.win-amd64-3.8\ddtrace\opentracer
  copying ddtrace\opentracer\settings.py -> build\lib.win-amd64-3.8\ddtrace\opentracer
  copying ddtrace\opentracer\span.py -> build\lib.win-amd64-3.8\ddtrace\opentracer
  copying ddtrace\opentracer\span_context.py -> build\lib.win-amd64-3.8\ddtrace\opentracer
  copying ddtrace\opentracer\tags.py -> build\lib.win-amd64-3.8\ddtrace\opentracer
  copying ddtrace\opentracer\tracer.py -> build\lib.win-amd64-3.8\ddtrace\opentracer
  copying ddtrace\opentracer\utils.py -> build\lib.win-amd64-3.8\ddtrace\opentracer
  copying ddtrace\opentracer\__init__.py -> build\lib.win-amd64-3.8\ddtrace\opentracer
  creating build\lib.win-amd64-3.8\ddtrace\profile
  copying ddtrace\profile\auto.py -> build\lib.win-amd64-3.8\ddtrace\profile
  copying ddtrace\profile\event.py -> build\lib.win-amd64-3.8\ddtrace\profile
  copying ddtrace\profile\profiler.py -> build\lib.win-amd64-3.8\ddtrace\profile
  copying ddtrace\profile\recorder.py -> build\lib.win-amd64-3.8\ddtrace\profile
  copying ddtrace\profile\scheduler.py -> build\lib.win-amd64-3.8\ddtrace\profile
  copying ddtrace\profile\__init__.py -> build\lib.win-amd64-3.8\ddtrace\profile
  creating build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\auto.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\event.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\profiler.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\recorder.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\scheduler.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\_attr.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\_line2def.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\_periodic.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\_service.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\_traceback.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\__init__.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  copying ddtrace\profiling\__main__.py -> build\lib.win-amd64-3.8\ddtrace\profiling
  creating build\lib.win-amd64-3.8\ddtrace\propagation
  copying ddtrace\propagation\http.py -> build\lib.win-amd64-3.8\ddtrace\propagation
  copying ddtrace\propagation\utils.py -> build\lib.win-amd64-3.8\ddtrace\propagation
  copying ddtrace\propagation\__init__.py -> build\lib.win-amd64-3.8\ddtrace\propagation
  creating build\lib.win-amd64-3.8\ddtrace\settings
  copying ddtrace\settings\config.py -> build\lib.win-amd64-3.8\ddtrace\settings
  copying ddtrace\settings\exceptions.py -> build\lib.win-amd64-3.8\ddtrace\settings
  copying ddtrace\settings\http.py -> build\lib.win-amd64-3.8\ddtrace\settings
  copying ddtrace\settings\integration.py -> build\lib.win-amd64-3.8\ddtrace\settings
  copying ddtrace\settings\__init__.py -> build\lib.win-amd64-3.8\ddtrace\settings
  creating build\lib.win-amd64-3.8\ddtrace\utils
  copying ddtrace\utils\attrdict.py -> build\lib.win-amd64-3.8\ddtrace\utils
  copying ddtrace\utils\config.py -> build\lib.win-amd64-3.8\ddtrace\utils
  copying ddtrace\utils\deprecation.py -> build\lib.win-amd64-3.8\ddtrace\utils
  copying ddtrace\utils\formats.py -> build\lib.win-amd64-3.8\ddtrace\utils
  copying ddtrace\utils\hook.py -> build\lib.win-amd64-3.8\ddtrace\utils
  copying ddtrace\utils\http.py -> build\lib.win-amd64-3.8\ddtrace\utils
  copying ddtrace\utils\importlib.py -> build\lib.win-amd64-3.8\ddtrace\utils
  copying ddtrace\utils\time.py -> build\lib.win-amd64-3.8\ddtrace\utils
  copying ddtrace\utils\wrappers.py -> build\lib.win-amd64-3.8\ddtrace\utils
  copying ddtrace\utils\__init__.py -> build\lib.win-amd64-3.8\ddtrace\utils
  creating build\lib.win-amd64-3.8\ddtrace\vendor
  copying ddtrace\vendor\__init__.py -> build\lib.win-amd64-3.8\ddtrace\vendor
  creating build\lib.win-amd64-3.8\ddtrace\contrib\aiobotocore
  copying ddtrace\contrib\aiobotocore\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\aiobotocore
  copying ddtrace\contrib\aiobotocore\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\aiobotocore
  creating build\lib.win-amd64-3.8\ddtrace\contrib\aiohttp
  copying ddtrace\contrib\aiohttp\middlewares.py -> build\lib.win-amd64-3.8\ddtrace\contrib\aiohttp
  copying ddtrace\contrib\aiohttp\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\aiohttp
  copying ddtrace\contrib\aiohttp\template.py -> build\lib.win-amd64-3.8\ddtrace\contrib\aiohttp
  copying ddtrace\contrib\aiohttp\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\aiohttp
  creating build\lib.win-amd64-3.8\ddtrace\contrib\aiopg
  copying ddtrace\contrib\aiopg\connection.py -> build\lib.win-amd64-3.8\ddtrace\contrib\aiopg
  copying ddtrace\contrib\aiopg\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\aiopg
  copying ddtrace\contrib\aiopg\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\aiopg
  creating build\lib.win-amd64-3.8\ddtrace\contrib\algoliasearch
  copying ddtrace\contrib\algoliasearch\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\algoliasearch
  copying ddtrace\contrib\algoliasearch\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\algoliasearch
  creating build\lib.win-amd64-3.8\ddtrace\contrib\asyncio
  copying ddtrace\contrib\asyncio\compat.py -> build\lib.win-amd64-3.8\ddtrace\contrib\asyncio
  copying ddtrace\contrib\asyncio\helpers.py -> build\lib.win-amd64-3.8\ddtrace\contrib\asyncio
  copying ddtrace\contrib\asyncio\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\asyncio
  copying ddtrace\contrib\asyncio\provider.py -> build\lib.win-amd64-3.8\ddtrace\contrib\asyncio
  copying ddtrace\contrib\asyncio\wrappers.py -> build\lib.win-amd64-3.8\ddtrace\contrib\asyncio
  copying ddtrace\contrib\asyncio\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\asyncio
  creating build\lib.win-amd64-3.8\ddtrace\contrib\boto
  copying ddtrace\contrib\boto\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\boto
  copying ddtrace\contrib\boto\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\boto
  creating build\lib.win-amd64-3.8\ddtrace\contrib\botocore
  copying ddtrace\contrib\botocore\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\botocore
  copying ddtrace\contrib\botocore\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\botocore
  creating build\lib.win-amd64-3.8\ddtrace\contrib\bottle
  copying ddtrace\contrib\bottle\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\bottle
  copying ddtrace\contrib\bottle\trace.py -> build\lib.win-amd64-3.8\ddtrace\contrib\bottle
  copying ddtrace\contrib\bottle\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\bottle
  creating build\lib.win-amd64-3.8\ddtrace\contrib\cassandra
  copying ddtrace\contrib\cassandra\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\cassandra
  copying ddtrace\contrib\cassandra\session.py -> build\lib.win-amd64-3.8\ddtrace\contrib\cassandra
  copying ddtrace\contrib\cassandra\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\cassandra
  creating build\lib.win-amd64-3.8\ddtrace\contrib\celery
  copying ddtrace\contrib\celery\app.py -> build\lib.win-amd64-3.8\ddtrace\contrib\celery
  copying ddtrace\contrib\celery\constants.py -> build\lib.win-amd64-3.8\ddtrace\contrib\celery
  copying ddtrace\contrib\celery\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\celery
  copying ddtrace\contrib\celery\signals.py -> build\lib.win-amd64-3.8\ddtrace\contrib\celery
  copying ddtrace\contrib\celery\task.py -> build\lib.win-amd64-3.8\ddtrace\contrib\celery
  copying ddtrace\contrib\celery\utils.py -> build\lib.win-amd64-3.8\ddtrace\contrib\celery
  copying ddtrace\contrib\celery\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\celery
  creating build\lib.win-amd64-3.8\ddtrace\contrib\consul
  copying ddtrace\contrib\consul\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\consul
  copying ddtrace\contrib\consul\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\consul
  creating build\lib.win-amd64-3.8\ddtrace\contrib\dbapi
  copying ddtrace\contrib\dbapi\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\dbapi
  creating build\lib.win-amd64-3.8\ddtrace\contrib\django
  copying ddtrace\contrib\django\apps.py -> build\lib.win-amd64-3.8\ddtrace\contrib\django
  copying ddtrace\contrib\django\compat.py -> build\lib.win-amd64-3.8\ddtrace\contrib\django
  copying ddtrace\contrib\django\conf.py -> build\lib.win-amd64-3.8\ddtrace\contrib\django
  copying ddtrace\contrib\django\middleware.py -> build\lib.win-amd64-3.8\ddtrace\contrib\django
  copying ddtrace\contrib\django\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\django
  copying ddtrace\contrib\django\restframework.py -> build\lib.win-amd64-3.8\ddtrace\contrib\django
  copying ddtrace\contrib\django\utils.py -> build\lib.win-amd64-3.8\ddtrace\contrib\django
  copying ddtrace\contrib\django\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\django
  creating build\lib.win-amd64-3.8\ddtrace\contrib\dogpile_cache
  copying ddtrace\contrib\dogpile_cache\lock.py -> build\lib.win-amd64-3.8\ddtrace\contrib\dogpile_cache
  copying ddtrace\contrib\dogpile_cache\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\dogpile_cache
  copying ddtrace\contrib\dogpile_cache\region.py -> build\lib.win-amd64-3.8\ddtrace\contrib\dogpile_cache
  copying ddtrace\contrib\dogpile_cache\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\dogpile_cache
  creating build\lib.win-amd64-3.8\ddtrace\contrib\elasticsearch
  copying ddtrace\contrib\elasticsearch\elasticsearch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\elasticsearch
  copying ddtrace\contrib\elasticsearch\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\elasticsearch
  copying ddtrace\contrib\elasticsearch\quantize.py -> build\lib.win-amd64-3.8\ddtrace\contrib\elasticsearch
  copying ddtrace\contrib\elasticsearch\transport.py -> build\lib.win-amd64-3.8\ddtrace\contrib\elasticsearch
  copying ddtrace\contrib\elasticsearch\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\elasticsearch
  creating build\lib.win-amd64-3.8\ddtrace\contrib\falcon
  copying ddtrace\contrib\falcon\middleware.py -> build\lib.win-amd64-3.8\ddtrace\contrib\falcon
  copying ddtrace\contrib\falcon\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\falcon
  copying ddtrace\contrib\falcon\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\falcon
  creating build\lib.win-amd64-3.8\ddtrace\contrib\flask
  copying ddtrace\contrib\flask\helpers.py -> build\lib.win-amd64-3.8\ddtrace\contrib\flask
  copying ddtrace\contrib\flask\middleware.py -> build\lib.win-amd64-3.8\ddtrace\contrib\flask
  copying ddtrace\contrib\flask\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\flask
  copying ddtrace\contrib\flask\wrappers.py -> build\lib.win-amd64-3.8\ddtrace\contrib\flask
  copying ddtrace\contrib\flask\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\flask
  creating build\lib.win-amd64-3.8\ddtrace\contrib\flask_cache
  copying ddtrace\contrib\flask_cache\tracers.py -> build\lib.win-amd64-3.8\ddtrace\contrib\flask_cache
  copying ddtrace\contrib\flask_cache\utils.py -> build\lib.win-amd64-3.8\ddtrace\contrib\flask_cache
  copying ddtrace\contrib\flask_cache\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\flask_cache
  creating build\lib.win-amd64-3.8\ddtrace\contrib\futures
  copying ddtrace\contrib\futures\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\futures
  copying ddtrace\contrib\futures\threading.py -> build\lib.win-amd64-3.8\ddtrace\contrib\futures
  copying ddtrace\contrib\futures\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\futures
  creating build\lib.win-amd64-3.8\ddtrace\contrib\gevent
  copying ddtrace\contrib\gevent\greenlet.py -> build\lib.win-amd64-3.8\ddtrace\contrib\gevent
  copying ddtrace\contrib\gevent\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\gevent
  copying ddtrace\contrib\gevent\provider.py -> build\lib.win-amd64-3.8\ddtrace\contrib\gevent
  copying ddtrace\contrib\gevent\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\gevent
  creating build\lib.win-amd64-3.8\ddtrace\contrib\grpc
  copying ddtrace\contrib\grpc\client_interceptor.py -> build\lib.win-amd64-3.8\ddtrace\contrib\grpc
  copying ddtrace\contrib\grpc\constants.py -> build\lib.win-amd64-3.8\ddtrace\contrib\grpc
  copying ddtrace\contrib\grpc\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\grpc
  copying ddtrace\contrib\grpc\server_interceptor.py -> build\lib.win-amd64-3.8\ddtrace\contrib\grpc
  copying ddtrace\contrib\grpc\utils.py -> build\lib.win-amd64-3.8\ddtrace\contrib\grpc
  copying ddtrace\contrib\grpc\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\grpc
  creating build\lib.win-amd64-3.8\ddtrace\contrib\httplib
  copying ddtrace\contrib\httplib\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\httplib
  copying ddtrace\contrib\httplib\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\httplib
  creating build\lib.win-amd64-3.8\ddtrace\contrib\jinja2
  copying ddtrace\contrib\jinja2\constants.py -> build\lib.win-amd64-3.8\ddtrace\contrib\jinja2
  copying ddtrace\contrib\jinja2\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\jinja2
  copying ddtrace\contrib\jinja2\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\jinja2
  creating build\lib.win-amd64-3.8\ddtrace\contrib\kombu
  copying ddtrace\contrib\kombu\constants.py -> build\lib.win-amd64-3.8\ddtrace\contrib\kombu
  copying ddtrace\contrib\kombu\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\kombu
  copying ddtrace\contrib\kombu\utils.py -> build\lib.win-amd64-3.8\ddtrace\contrib\kombu
  copying ddtrace\contrib\kombu\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\kombu
  creating build\lib.win-amd64-3.8\ddtrace\contrib\logging
  copying ddtrace\contrib\logging\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\logging
  copying ddtrace\contrib\logging\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\logging
  creating build\lib.win-amd64-3.8\ddtrace\contrib\mako
  copying ddtrace\contrib\mako\constants.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mako
  copying ddtrace\contrib\mako\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mako
  copying ddtrace\contrib\mako\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mako
  creating build\lib.win-amd64-3.8\ddtrace\contrib\molten
  copying ddtrace\contrib\molten\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\molten
  copying ddtrace\contrib\molten\wrappers.py -> build\lib.win-amd64-3.8\ddtrace\contrib\molten
  copying ddtrace\contrib\molten\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\molten
  creating build\lib.win-amd64-3.8\ddtrace\contrib\mongoengine
  copying ddtrace\contrib\mongoengine\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mongoengine
  copying ddtrace\contrib\mongoengine\trace.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mongoengine
  copying ddtrace\contrib\mongoengine\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mongoengine
  creating build\lib.win-amd64-3.8\ddtrace\contrib\mysql
  copying ddtrace\contrib\mysql\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mysql
  copying ddtrace\contrib\mysql\tracers.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mysql
  copying ddtrace\contrib\mysql\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mysql
  creating build\lib.win-amd64-3.8\ddtrace\contrib\mysqldb
  copying ddtrace\contrib\mysqldb\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mysqldb
  copying ddtrace\contrib\mysqldb\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\mysqldb
  creating build\lib.win-amd64-3.8\ddtrace\contrib\psycopg
  copying ddtrace\contrib\psycopg\connection.py -> build\lib.win-amd64-3.8\ddtrace\contrib\psycopg
  copying ddtrace\contrib\psycopg\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\psycopg
  copying ddtrace\contrib\psycopg\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\psycopg
  creating build\lib.win-amd64-3.8\ddtrace\contrib\pylibmc
  copying ddtrace\contrib\pylibmc\addrs.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pylibmc
  copying ddtrace\contrib\pylibmc\client.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pylibmc
  copying ddtrace\contrib\pylibmc\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pylibmc
  copying ddtrace\contrib\pylibmc\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pylibmc
  creating build\lib.win-amd64-3.8\ddtrace\contrib\pylons
  copying ddtrace\contrib\pylons\compat.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pylons
  copying ddtrace\contrib\pylons\constants.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pylons
  copying ddtrace\contrib\pylons\middleware.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pylons
  copying ddtrace\contrib\pylons\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pylons
  copying ddtrace\contrib\pylons\renderer.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pylons
  copying ddtrace\contrib\pylons\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pylons
  creating build\lib.win-amd64-3.8\ddtrace\contrib\pymemcache
  copying ddtrace\contrib\pymemcache\client.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pymemcache
  copying ddtrace\contrib\pymemcache\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pymemcache
  copying ddtrace\contrib\pymemcache\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pymemcache
  creating build\lib.win-amd64-3.8\ddtrace\contrib\pymongo
  copying ddtrace\contrib\pymongo\client.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pymongo
  copying ddtrace\contrib\pymongo\parse.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pymongo
  copying ddtrace\contrib\pymongo\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pymongo
  copying ddtrace\contrib\pymongo\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pymongo
  creating build\lib.win-amd64-3.8\ddtrace\contrib\pymysql
  copying ddtrace\contrib\pymysql\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pymysql
  copying ddtrace\contrib\pymysql\tracers.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pymysql
  copying ddtrace\contrib\pymysql\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pymysql
  creating build\lib.win-amd64-3.8\ddtrace\contrib\pyramid
  copying ddtrace\contrib\pyramid\constants.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pyramid
  copying ddtrace\contrib\pyramid\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pyramid
  copying ddtrace\contrib\pyramid\trace.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pyramid
  copying ddtrace\contrib\pyramid\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\pyramid
  creating build\lib.win-amd64-3.8\ddtrace\contrib\redis
  copying ddtrace\contrib\redis\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\redis
  copying ddtrace\contrib\redis\tracers.py -> build\lib.win-amd64-3.8\ddtrace\contrib\redis
  copying ddtrace\contrib\redis\util.py -> build\lib.win-amd64-3.8\ddtrace\contrib\redis
  copying ddtrace\contrib\redis\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\redis
  creating build\lib.win-amd64-3.8\ddtrace\contrib\rediscluster
  copying ddtrace\contrib\rediscluster\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\rediscluster
  copying ddtrace\contrib\rediscluster\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\rediscluster
  creating build\lib.win-amd64-3.8\ddtrace\contrib\requests
  copying ddtrace\contrib\requests\connection.py -> build\lib.win-amd64-3.8\ddtrace\contrib\requests
  copying ddtrace\contrib\requests\constants.py -> build\lib.win-amd64-3.8\ddtrace\contrib\requests
  copying ddtrace\contrib\requests\legacy.py -> build\lib.win-amd64-3.8\ddtrace\contrib\requests
  copying ddtrace\contrib\requests\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\requests
  copying ddtrace\contrib\requests\session.py -> build\lib.win-amd64-3.8\ddtrace\contrib\requests
  copying ddtrace\contrib\requests\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\requests
  creating build\lib.win-amd64-3.8\ddtrace\contrib\sqlalchemy
  copying ddtrace\contrib\sqlalchemy\engine.py -> build\lib.win-amd64-3.8\ddtrace\contrib\sqlalchemy
  copying ddtrace\contrib\sqlalchemy\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\sqlalchemy
  copying ddtrace\contrib\sqlalchemy\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\sqlalchemy
  creating build\lib.win-amd64-3.8\ddtrace\contrib\sqlite3
  copying ddtrace\contrib\sqlite3\connection.py -> build\lib.win-amd64-3.8\ddtrace\contrib\sqlite3
  copying ddtrace\contrib\sqlite3\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\sqlite3
  copying ddtrace\contrib\sqlite3\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\sqlite3
  creating build\lib.win-amd64-3.8\ddtrace\contrib\tornado
  copying ddtrace\contrib\tornado\application.py -> build\lib.win-amd64-3.8\ddtrace\contrib\tornado
  copying ddtrace\contrib\tornado\compat.py -> build\lib.win-amd64-3.8\ddtrace\contrib\tornado
  copying ddtrace\contrib\tornado\constants.py -> build\lib.win-amd64-3.8\ddtrace\contrib\tornado
  copying ddtrace\contrib\tornado\decorators.py -> build\lib.win-amd64-3.8\ddtrace\contrib\tornado
  copying ddtrace\contrib\tornado\handlers.py -> build\lib.win-amd64-3.8\ddtrace\contrib\tornado
  copying ddtrace\contrib\tornado\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\tornado
  copying ddtrace\contrib\tornado\stack_context.py -> build\lib.win-amd64-3.8\ddtrace\contrib\tornado
  copying ddtrace\contrib\tornado\template.py -> build\lib.win-amd64-3.8\ddtrace\contrib\tornado
  copying ddtrace\contrib\tornado\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\tornado
  creating build\lib.win-amd64-3.8\ddtrace\contrib\vertica
  copying ddtrace\contrib\vertica\constants.py -> build\lib.win-amd64-3.8\ddtrace\contrib\vertica
  copying ddtrace\contrib\vertica\patch.py -> build\lib.win-amd64-3.8\ddtrace\contrib\vertica
  copying ddtrace\contrib\vertica\__init__.py -> build\lib.win-amd64-3.8\ddtrace\contrib\vertica
  creating build\lib.win-amd64-3.8\ddtrace\internal\runtime
  copying ddtrace\internal\runtime\collector.py -> build\lib.win-amd64-3.8\ddtrace\internal\runtime
  copying ddtrace\internal\runtime\constants.py -> build\lib.win-amd64-3.8\ddtrace\internal\runtime
  copying ddtrace\internal\runtime\container.py -> build\lib.win-amd64-3.8\ddtrace\internal\runtime
  copying ddtrace\internal\runtime\metric_collectors.py -> build\lib.win-amd64-3.8\ddtrace\internal\runtime
  copying ddtrace\internal\runtime\runtime_metrics.py -> build\lib.win-amd64-3.8\ddtrace\internal\runtime
  copying ddtrace\internal\runtime\tag_collectors.py -> build\lib.win-amd64-3.8\ddtrace\internal\runtime
  copying ddtrace\internal\runtime\__init__.py -> build\lib.win-amd64-3.8\ddtrace\internal\runtime
  creating build\lib.win-amd64-3.8\ddtrace\opentracer\propagation
  copying ddtrace\opentracer\propagation\binary.py -> build\lib.win-amd64-3.8\ddtrace\opentracer\propagation
  copying ddtrace\opentracer\propagation\http.py -> build\lib.win-amd64-3.8\ddtrace\opentracer\propagation
  copying ddtrace\opentracer\propagation\propagator.py -> build\lib.win-amd64-3.8\ddtrace\opentracer\propagation
  copying ddtrace\opentracer\propagation\text.py -> build\lib.win-amd64-3.8\ddtrace\opentracer\propagation
  copying ddtrace\opentracer\propagation\__init__.py -> build\lib.win-amd64-3.8\ddtrace\opentracer\propagation
  creating build\lib.win-amd64-3.8\ddtrace\profile\bootstrap
  copying ddtrace\profile\bootstrap\sitecustomize.py -> build\lib.win-amd64-3.8\ddtrace\profile\bootstrap
  copying ddtrace\profile\bootstrap\__init__.py -> build\lib.win-amd64-3.8\ddtrace\profile\bootstrap
  creating build\lib.win-amd64-3.8\ddtrace\profile\collector
  copying ddtrace\profile\collector\exceptions.py -> build\lib.win-amd64-3.8\ddtrace\profile\collector
  copying ddtrace\profile\collector\memory.py -> build\lib.win-amd64-3.8\ddtrace\profile\collector
  copying ddtrace\profile\collector\stack.py -> build\lib.win-amd64-3.8\ddtrace\profile\collector
  copying ddtrace\profile\collector\threading.py -> build\lib.win-amd64-3.8\ddtrace\profile\collector
  copying ddtrace\profile\collector\__init__.py -> build\lib.win-amd64-3.8\ddtrace\profile\collector
  creating build\lib.win-amd64-3.8\ddtrace\profile\exporter
  copying ddtrace\profile\exporter\file.py -> build\lib.win-amd64-3.8\ddtrace\profile\exporter
  copying ddtrace\profile\exporter\http.py -> build\lib.win-amd64-3.8\ddtrace\profile\exporter
  copying ddtrace\profile\exporter\pprof.py -> build\lib.win-amd64-3.8\ddtrace\profile\exporter
  copying ddtrace\profile\exporter\pprof_pb2.py -> build\lib.win-amd64-3.8\ddtrace\profile\exporter
  copying ddtrace\profile\exporter\__init__.py -> build\lib.win-amd64-3.8\ddtrace\profile\exporter
  creating build\lib.win-amd64-3.8\ddtrace\profiling\bootstrap
  copying ddtrace\profiling\bootstrap\sitecustomize.py -> build\lib.win-amd64-3.8\ddtrace\profiling\bootstrap
  copying ddtrace\profiling\bootstrap\__init__.py -> build\lib.win-amd64-3.8\ddtrace\profiling\bootstrap
  creating build\lib.win-amd64-3.8\ddtrace\profiling\collector
  copying ddtrace\profiling\collector\exceptions.py -> build\lib.win-amd64-3.8\ddtrace\profiling\collector
  copying ddtrace\profiling\collector\memory.py -> build\lib.win-amd64-3.8\ddtrace\profiling\collector
  copying ddtrace\profiling\collector\threading.py -> build\lib.win-amd64-3.8\ddtrace\profiling\collector
  copying ddtrace\profiling\collector\__init__.py -> build\lib.win-amd64-3.8\ddtrace\profiling\collector
  creating build\lib.win-amd64-3.8\ddtrace\profiling\exporter
  copying ddtrace\profiling\exporter\file.py -> build\lib.win-amd64-3.8\ddtrace\profiling\exporter
  copying ddtrace\profiling\exporter\http.py -> build\lib.win-amd64-3.8\ddtrace\profiling\exporter
  copying ddtrace\profiling\exporter\pprof.py -> build\lib.win-amd64-3.8\ddtrace\profiling\exporter
  copying ddtrace\profiling\exporter\pprof_pb2.py -> build\lib.win-amd64-3.8\ddtrace\profiling\exporter
  copying ddtrace\profiling\exporter\__init__.py -> build\lib.win-amd64-3.8\ddtrace\profiling\exporter
  creating build\lib.win-amd64-3.8\ddtrace\vendor\attr
  copying ddtrace\vendor\attr\converters.py -> build\lib.win-amd64-3.8\ddtrace\vendor\attr
  copying ddtrace\vendor\attr\exceptions.py -> build\lib.win-amd64-3.8\ddtrace\vendor\attr
  copying ddtrace\vendor\attr\filters.py -> build\lib.win-amd64-3.8\ddtrace\vendor\attr
  copying ddtrace\vendor\attr\validators.py -> build\lib.win-amd64-3.8\ddtrace\vendor\attr
  copying ddtrace\vendor\attr\_compat.py -> build\lib.win-amd64-3.8\ddtrace\vendor\attr
  copying ddtrace\vendor\attr\_config.py -> build\lib.win-amd64-3.8\ddtrace\vendor\attr
  copying ddtrace\vendor\attr\_funcs.py -> build\lib.win-amd64-3.8\ddtrace\vendor\attr
  copying ddtrace\vendor\attr\_make.py -> build\lib.win-amd64-3.8\ddtrace\vendor\attr
  copying ddtrace\vendor\attr\_version_info.py -> build\lib.win-amd64-3.8\ddtrace\vendor\attr
  copying ddtrace\vendor\attr\__init__.py -> build\lib.win-amd64-3.8\ddtrace\vendor\attr
  creating build\lib.win-amd64-3.8\ddtrace\vendor\debtcollector
  copying ddtrace\vendor\debtcollector\moves.py -> build\lib.win-amd64-3.8\ddtrace\vendor\debtcollector
  copying ddtrace\vendor\debtcollector\removals.py -> build\lib.win-amd64-3.8\ddtrace\vendor\debtcollector
  copying ddtrace\vendor\debtcollector\renames.py -> build\lib.win-amd64-3.8\ddtrace\vendor\debtcollector
  copying ddtrace\vendor\debtcollector\updating.py -> build\lib.win-amd64-3.8\ddtrace\vendor\debtcollector
  copying ddtrace\vendor\debtcollector\_utils.py -> build\lib.win-amd64-3.8\ddtrace\vendor\debtcollector
  copying ddtrace\vendor\debtcollector\__init__.py -> build\lib.win-amd64-3.8\ddtrace\vendor\debtcollector
  creating build\lib.win-amd64-3.8\ddtrace\vendor\dogstatsd
  copying ddtrace\vendor\dogstatsd\base.py -> build\lib.win-amd64-3.8\ddtrace\vendor\dogstatsd
  copying ddtrace\vendor\dogstatsd\compat.py -> build\lib.win-amd64-3.8\ddtrace\vendor\dogstatsd
  copying ddtrace\vendor\dogstatsd\context.py -> build\lib.win-amd64-3.8\ddtrace\vendor\dogstatsd
  copying ddtrace\vendor\dogstatsd\context_async.py -> build\lib.win-amd64-3.8\ddtrace\vendor\dogstatsd
  copying ddtrace\vendor\dogstatsd\route.py -> build\lib.win-amd64-3.8\ddtrace\vendor\dogstatsd
  copying ddtrace\vendor\dogstatsd\__init__.py -> build\lib.win-amd64-3.8\ddtrace\vendor\dogstatsd
  creating build\lib.win-amd64-3.8\ddtrace\vendor\monotonic
  copying ddtrace\vendor\monotonic\__init__.py -> build\lib.win-amd64-3.8\ddtrace\vendor\monotonic
  creating build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\setup.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\_common.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\_compat.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\_psaix.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\_psbsd.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\_pslinux.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\_psosx.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\_psposix.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\_pssunos.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\_pswindows.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  copying ddtrace\vendor\psutil\__init__.py -> build\lib.win-amd64-3.8\ddtrace\vendor\psutil
  creating build\lib.win-amd64-3.8\ddtrace\vendor\six
  copying ddtrace\vendor\six\__init__.py -> build\lib.win-amd64-3.8\ddtrace\vendor\six
  creating build\lib.win-amd64-3.8\ddtrace\vendor\wrapt
  copying ddtrace\vendor\wrapt\decorators.py -> build\lib.win-amd64-3.8\ddtrace\vendor\wrapt
  copying ddtrace\vendor\wrapt\importer.py -> build\lib.win-amd64-3.8\ddtrace\vendor\wrapt
  copying ddtrace\vendor\wrapt\setup.py -> build\lib.win-amd64-3.8\ddtrace\vendor\wrapt
  copying ddtrace\vendor\wrapt\wrappers.py -> build\lib.win-amd64-3.8\ddtrace\vendor\wrapt
  copying ddtrace\vendor\wrapt\__init__.py -> build\lib.win-amd64-3.8\ddtrace\vendor\wrapt
  running build_ext
  building 'ddtrace.internal._rand' extension
  creating build\temp.win-amd64-3.8
  creating build\temp.win-amd64-3.8\Release
  creating build\temp.win-amd64-3.8\Release\ddtrace
  creating build\temp.win-amd64-3.8\Release\ddtrace\internal
  C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.26.28801\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\Users\opiot\venv\dd\include -IC:\Users\opiot\AppData\Local\Programs\Python\Python38\include -IC:\Users\opiot\AppData\Local\Programs\Python\Python38\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.26.28801\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcddtrace/internal/_rand.c /Fobuild\temp.win-amd64-3.8\Release\ddtrace/internal/_rand.obj
  _rand.c
  C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.26.28801\bin\HostX86\x64\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\Users\opiot\venv\dd\libs /LIBPATH:C:\Users\opiot\AppData\Local\Programs\Python\Python38\libs /LIBPATH:C:\Users\opiot\AppData\Local\Programs\Python\Python38 /LIBPATH:C:\Users\opiot\venv\dd\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.26.28801\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\um\x64" /EXPORT:PyInit__rand build\temp.win-amd64-3.8\Release\ddtrace/internal/_rand.obj /OUT:build\lib.win-amd64-3.8\ddtrace\internal\_rand.cp38-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.8\Release\ddtrace/internal\_rand.cp38-win_amd64.lib
     Cr‚ation de la bibliothŠque build\temp.win-amd64-3.8\Release\ddtrace/internal\_rand.cp38-win_amd64.lib et de l'objet build\temp.win-amd64-3.8\Release\ddtrace/internal\_rand.cp38-win_amd64.exp
  G‚n‚ration de code en cours
  Fin de la g‚n‚ration du code
  building 'ddtrace.profiling.collector.stack' extension
  creating build\temp.win-amd64-3.8\Release\ddtrace\profiling
  creating build\temp.win-amd64-3.8\Release\ddtrace\profiling\collector
  C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.26.28801\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\Users\opiot\venv\dd\include -IC:\Users\opiot\AppData\Local\Programs\Python\Python38\include -IC:\Users\opiot\AppData\Local\Programs\Python\Python38\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.26.28801\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcddtrace/profiling/collector/stack.c /Fobuild\temp.win-amd64-3.8\Release\ddtrace/profiling/collector/stack.obj -DPy_BUILD_CORE
  stack.c
  C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.26.28801\bin\HostX86\x64\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\Users\opiot\venv\dd\libs /LIBPATH:C:\Users\opiot\AppData\Local\Programs\Python\Python38\libs /LIBPATH:C:\Users\opiot\AppData\Local\Programs\Python\Python38 /LIBPATH:C:\Users\opiot\venv\dd\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.26.28801\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\um\x64" /EXPORT:PyInit_stack build\temp.win-amd64-3.8\Release\ddtrace/profiling/collector/stack.obj /OUT:build\lib.win-amd64-3.8\ddtrace\profiling\collector\stack.cp38-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.8\Release\ddtrace/profiling/collector\stack.cp38-win_amd64.lib
     Cr‚ation de la bibliothŠque build\temp.win-amd64-3.8\Release\ddtrace/profiling/collector\stack.cp38-win_amd64.lib et de l'objet build\temp.win-amd64-3.8\Release\ddtrace/profiling/collector\stack.cp38-win_amd64.exp
  stack.obj : error LNK2001: symbole externe non r‚solu PyFrozenSet_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PySequence_List
  stack.obj : error LNK2001: symbole externe non r‚solu PyBaseObject_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_SetAttr
  stack.obj : error LNK2001: symbole externe non r‚solu PyUnicode_InternFromString
  stack.obj : error LNK2001: symbole externe non r‚solu PyCFunction_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyOS_snprintf
  stack.obj : error LNK2001: symbole externe non r‚solu PyThreadState_Get
  stack.obj : error LNK2001: symbole externe non r‚solu _Py_CheckRecursionLimit
  stack.obj : error LNK2001: symbole externe non r‚solu PyNumber_InPlaceTrueDivide
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_SetObject
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_GivenExceptionMatches
  stack.obj : error LNK2001: symbole externe non r‚solu _PyObject_CallFunction_SizeT
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_WarnEx
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_RuntimeWarning
  stack.obj : error LNK2001: symbole externe non r‚solu PyFunction_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyFrame_New
  stack.obj : error LNK2001: symbole externe non r‚solu PyLong_AsSsize_t
  stack.obj : error LNK2001: symbole externe non r‚solu _PyDict_SetItem_KnownHash
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_KeyError
  stack.obj : error LNK2001: symbole externe non r‚solu PyImport_ImportModule
  stack.obj : error LNK2001: symbole externe non r‚solu PyImport_ImportModuleLevelObject
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_NormalizeException
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_SelfIter
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_Occurred
  stack.obj : error LNK2001: symbole externe non r‚solu PyType_GetFlags
  stack.obj : error LNK2001: symbole externe non r‚solu PyLong_FromSsize_t
  stack.obj : error LNK2001: symbole externe non r‚solu PyNumber_Remainder
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_GenericGetAttr
  stack.obj : error LNK2001: symbole externe non r‚solu PyTraceBack_Here
  stack.obj : error LNK2001: symbole externe non r‚solu _PyThreadState_UncheckedGet
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_RuntimeError
  stack.obj : error LNK2001: symbole externe non r‚solu PyMethod_New
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_SetAttrString
  stack.obj : error LNK2001: symbole externe non r‚solu PyLong_FromLong
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_GC_UnTrack
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_Hash
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_SetNone
  stack.obj : error LNK2001: symbole externe non r‚solu PyInterpreterState_GetID
  stack.obj : error LNK2001: symbole externe non r‚solu PyLong_FromUnsignedLongLong
  stack.obj : error LNK2001: symbole externe non r‚solu Py_GetVersion
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_GetAttr
  stack.obj : error LNK2001: symbole externe non r‚solu PyRun_StringFlags
  stack.obj : error LNK2001: symbole externe non r‚solu _Py_NoneStruct
  stack.obj : error LNK2001: symbole externe non r‚solu PyTuple_New
  stack.obj : error LNK2001: symbole externe non r‚solu _PyErr_GetTopmostException
  stack.obj : error LNK2001: symbole externe non r‚solu PySequence_Contains
  stack.obj : error LNK2001: symbole externe non r‚solu PyDict_SetItemString
  stack.obj : error LNK2001: symbole externe non r‚solu PyDict_Size
  stack.obj : error LNK2001: symbole externe non r‚solu PyLong_FromLongLong
  stack.obj : error LNK2001: symbole externe non r‚solu PyThread_acquire_lock
  stack.obj : error LNK2001: symbole externe non r‚solu PyFrame_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyFloat_FromDouble
  stack.obj : error LNK2001: symbole externe non r‚solu PySet_Add
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_AttributeError
  stack.obj : error LNK2001: symbole externe non r‚solu PyTuple_GetSlice
  stack.obj : error LNK2001: symbole externe non r‚solu PySet_Size
  stack.obj : error LNK2001: symbole externe non r‚solu PyNumber_Multiply
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_SetString
  stack.obj : error LNK2001: symbole externe non r‚solu PyIter_Next
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_WriteUnraisable
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_GetIter
  stack.obj : error LNK2001: symbole externe non r‚solu PyEval_EvalCodeEx
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_ValueError
  stack.obj : error LNK2001: symbole externe non r‚solu _Py_CheckRecursiveCall
  stack.obj : error LNK2001: symbole externe non r‚solu PyLong_FromUnsignedLong
  stack.obj : error LNK2001: symbole externe non r‚solu PyDict_Next
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_Format
  stack.obj : error LNK2001: symbole externe non r‚solu PyDict_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyThreadState_Next
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_RichCompare
  stack.obj : error LNK2001: symbole externe non r‚solu PyTuple_Type
  stack.obj : error LNK2001: symbole externe non r‚solu _Py_FalseStruct
  stack.obj : error LNK2001: symbole externe non r‚solu PyCoro_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyFloat_Type
  stack.obj : error LNK2001: symbole externe non r‚solu _PyLong_FromByteArray
  stack.obj : error LNK2001: symbole externe non r‚solu PyModule_NewObject
  stack.obj : error LNK2001: symbole externe non r‚solu PyMethod_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyLong_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyType_IsSubtype
  stack.obj : error LNK2001: symbole externe non r‚solu PyNumber_Subtract
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_OverflowError
  stack.obj : error LNK2001: symbole externe non r‚solu _Py_Dealloc
  stack.obj : error LNK2001: symbole externe non r‚solu PyTuple_GetItem
  stack.obj : error LNK2001: symbole externe non r‚solu PyImport_GetModuleDict
  stack.obj : error LNK2001: symbole externe non r‚solu PyModule_GetDict
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_Free
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_ExceptionMatches
  stack.obj : error LNK2001: symbole externe non r‚solu PyThread_release_lock
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_GC_Del
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_CallFunctionObjArgs
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_ClearWeakRefs
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_Not
  stack.obj : error LNK2001: symbole externe non r‚solu PyUnicode_FromFormat
  stack.obj : error LNK2001: symbole externe non r‚solu PySet_New
  stack.obj : error LNK2001: symbole externe non r‚solu PyList_New
  stack.obj : error LNK2001: symbole externe non r‚solu PyImport_AddModule
  stack.obj : error LNK2001: symbole externe non r‚solu PyType_Ready
  stack.obj : error LNK2001: symbole externe non r‚solu PyInterpreterState_Next
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_GetAttrString
  stack.obj : error LNK2001: symbole externe non r‚solu _PyGen_Send
  stack.obj : error LNK2001: symbole externe non r‚solu PyErr_Clear
  stack.obj : error LNK2001: symbole externe non r‚solu PyList_Append
  stack.obj : error LNK2001: symbole externe non r‚solu PyUnicode_Decode
  stack.obj : error LNK2001: symbole externe non r‚solu PyLong_AsDouble
  stack.obj : error LNK2001: symbole externe non r‚solu PyCode_New
  stack.obj : error LNK2001: symbole externe non r‚solu PyException_SetTraceback
  stack.obj : error LNK2001: symbole externe non r‚solu PySet_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyDict_SetItem
  stack.obj : error LNK2001: symbole externe non r‚solu PyDict_New
  stack.obj : error LNK2001: symbole externe non r‚solu PyUnicode_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_CallFinalizerFromDealloc
  stack.obj : error LNK2001: symbole externe non r‚solu _PyDict_GetItem_KnownHash
  stack.obj : error LNK2001: symbole externe non r‚solu PyNumber_Index
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_StopIteration
  stack.obj : error LNK2001: symbole externe non r‚solu PyFrozenSet_New
  stack.obj : error LNK2001: symbole externe non r‚solu _PySet_NextEntry
  stack.obj : error LNK2001: symbole externe non r‚solu PyList_Type
  stack.obj : error LNK2001: symbole externe non r‚solu _PyRuntime
  stack.obj : error LNK2001: symbole externe non r‚solu PyDict_GetItemString
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_GetItem
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_CallObject
  stack.obj : error LNK2001: symbole externe non r‚solu PyModuleDef_Init
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_GC_Track
  stack.obj : error LNK2001: symbole externe non r‚solu PyBytes_FromStringAndSize
  stack.obj : error LNK2001: symbole externe non r‚solu PyNumber_Long
  stack.obj : error LNK2001: symbole externe non r‚solu PyUnicode_Compare
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_IsSubclass
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_TypeError
  stack.obj : error LNK2001: symbole externe non r‚solu PyMem_Realloc
  stack.obj : error LNK2001: symbole externe non r‚solu _Py_tracemalloc_config
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_IsTrue
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_NameError
  stack.obj : error LNK2001: symbole externe non r‚solu PyTuple_Pack
  stack.obj : error LNK2001: symbole externe non r‚solu PyMem_Malloc
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_IndexError
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_ImportError
  stack.obj : error LNK2001: symbole externe non r‚solu _PyTraceMalloc_NewReference
  stack.obj : error LNK2001: symbole externe non r‚solu PyGen_Type
  stack.obj : error LNK2001: symbole externe non r‚solu _Py_TrueStruct
  stack.obj : error LNK2001: symbole externe non r‚solu PyArg_UnpackTuple
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_SystemError
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_SetItem
  stack.obj : error LNK2001: symbole externe non r‚solu _PyObject_GC_New
  stack.obj : error LNK2001: symbole externe non r‚solu PyException_SetCause
  stack.obj : error LNK2001: symbole externe non r‚solu PyNumber_TrueDivide
  stack.obj : error LNK2001: symbole externe non r‚solu PyTraceBack_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyMethodDescr_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PyExc_GeneratorExit
  stack.obj : error LNK2001: symbole externe non r‚solu PyInterpreterState_ThreadHead
  stack.obj : error LNK2001: symbole externe non r‚solu PyEval_EvalFrameEx
  stack.obj : error LNK2001: symbole externe non r‚solu PyUnicode_FromString
  stack.obj : error LNK2001: symbole externe non r‚solu PyNumber_InPlaceFloorDivide
  stack.obj : error LNK2001: symbole externe non r‚solu _PyType_Lookup
  stack.obj : error LNK2001: symbole externe non r‚solu PyUnicode_Format
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_Size
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_Call
  stack.obj : error LNK2001: symbole externe non r‚solu PyNumber_InPlaceSubtract
  stack.obj : error LNK2001: symbole externe non r‚solu PyType_Type
  stack.obj : error LNK2001: symbole externe non r‚solu PySet_Discard
  stack.obj : error LNK2001: symbole externe non r‚solu PyInterpreterState_Head
  stack.obj : error LNK2001: symbole externe non r‚solu PyObject_DelItem
  stack.obj : error LNK2001: symbole externe non r‚solu PyUnicode_FromStringAndSize
  stack.obj : error LNK2001: symbole externe non r‚solu _PyObject_GetDictPtr
  build\lib.win-amd64-3.8\ddtrace\profiling\collector\stack.cp38-win_amd64.pyd : fatal error LNK1120: 164 externes non r‚solus
  C:\Users\opiot\AppData\Local\Temp\pip-build-env-5qgkbhbq\overlay\Lib\site-packages\Cython\Compiler\Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: C:\Users\opiot\AppData\Local\Temp\pip-install-t1aocqdk\ddtrace\ddtrace\internal\_rand.pyx
    tree = Parsing.p_module(s, pxd, full_module_name)
  C:\Users\opiot\AppData\Local\Temp\pip-build-env-5qgkbhbq\overlay\Lib\site-packages\Cython\Compiler\Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: C:\Users\opiot\AppData\Local\Temp\pip-install-t1aocqdk\ddtrace\ddtrace\profiling\_build.pyx
    tree = Parsing.p_module(s, pxd, full_module_name)
  C:\Users\opiot\AppData\Local\Temp\pip-build-env-5qgkbhbq\overlay\Lib\site-packages\Cython\Compiler\Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: C:\Users\opiot\AppData\Local\Temp\pip-install-t1aocqdk\ddtrace\ddtrace\profiling\collector\_traceback.pyx
    tree = Parsing.p_module(s, pxd, full_module_name)
  C:\Users\opiot\AppData\Local\Temp\pip-build-env-5qgkbhbq\overlay\Lib\site-packages\Cython\Compiler\Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: C:\Users\opiot\AppData\Local\Temp\pip-install-t1aocqdk\ddtrace\ddtrace\profiling\collector\stack.pyx
    tree = Parsing.p_module(s, pxd, full_module_name)

  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Tools\\MSVC\\14.26.28801\\bin\\HostX86\\x64\\link.exe' failed with exit status 1120
  ----------------------------------------

  ERROR: Failed building wheel for ddtrace
Failed to build ddtrace
ERROR: Could not build wheels for ddtrace which use PEP 517 and cannot be installed directly
```



The cythonize `ddtrace/profiling/collector/stack.pyx` cannot be compiled.

I have tried to add Cython directive 'language_level' to 3 in `setup.py` but the compilation still failed.

```python
    use_scm_version=True,
    setup_requires=["setuptools_scm", "cython"],
    ext_modules=cythonize(
        [
            Cython.Distutils.Extension(
                "ddtrace.internal._rand", sources=["ddtrace/internal/_rand.pyx"], language="c",
            ),
            Cython.Distutils.Extension(
                "ddtrace.profiling.collector.stack",
                sources=["ddtrace/profiling/collector/stack.pyx"],
                language="c",
                extra_compile_args=["-DPy_BUILD_CORE"],
            ),
            Cython.Distutils.Extension(
                "ddtrace.profiling.collector._traceback",
                sources=["ddtrace/profiling/collector/_traceback.pyx"],
                language="c",
            ),
            Cython.Distutils.Extension(
                "ddtrace.profiling._build", sources=["ddtrace/profiling/_build.pyx"], language="c",
            ),
        ],
        compiler_directives={'language_level' : "3"},
        compile_time_env={
            "PY_MAJOR_VERSION": sys.version_info.major,
            "PY_MINOR_VERSION": sys.version_info.minor,
            "PY_MICRO_VERSION": sys.version_info.micro,
        },
        force=True,
    )
```



For information with ddtrace 0.36.1 is working fine (the issue start with 0.37.0)

```
(dd) PS C:\Users\opiot> python -m pip install ddtrace==0.36.1
Processing c:\users\opiot\appdata\local\pip\cache\wheels\85\c3\ee\11aeab9405c28d6651d517554e7e4d43fa7e688c42453c4aed\ddtrace-0.36.1-cp38-cp38-win_amd64.whl
Collecting msgpack>=0.5.0
  Using cached msgpack-1.0.0-cp38-cp38-win_amd64.whl (73 kB)
Installing collected packages: msgpack, ddtrace
Successfully installed ddtrace-0.36.1 msgpack-1.0.0
```

--- a/ddtrace/profiling/collector/stack.pyx
+++ b/ddtrace/profiling/collector/stack.pyx
@@ -172,7 +172,7 @@ class StackExceptionSampleEvent(StackBasedEvent):
 
 
 # The head lock (the interpreter mutex) is only exposed in a data structure in Python ≥ 3.7
-IF PY_MAJOR_VERSION >= 3 and PY_MINOR_VERSION >= 7:
+IF UNAME_SYSNAME != "Windows" and PY_MAJOR_VERSION >= 3 and PY_MINOR_VERSION >= 7:
     FEATURES['stack-exceptions'] = True
 
     from cpython cimport PyInterpreterState
@@ -250,7 +250,7 @@ cdef get_thread_name(thread_id):
 cdef stack_collect(ignore_profiler, thread_time, max_nframes, interval, wall_time, thread_span_links):
     current_exceptions = []
 
-    IF PY_MAJOR_VERSION >= 3 and PY_MINOR_VERSION >= 7:
+    IF UNAME_SYSNAME != "Windows" and PY_MAJOR_VERSION >= 3 and PY_MINOR_VERSION >= 7:
         cdef PyInterpreterState* interp
         cdef PyThreadState* tstate
         cdef _PyErr_StackItem* exc_info

--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,5 @@
 import os
+import platform
 import sys
 
 from setuptools import setup, find_packages
@@ -100,6 +101,12 @@ def get_exts_for(name):
         return []
 
 
+if platform.uname()[0] != "Windows":
+    extra_compile_args = ["-DPy_BUILD_CORE"]
+else:
+    extra_compile_args = []
+
+
 # Base `setup()` kwargs without any C-extension registering
 setup(
     **dict(
@@ -157,7 +164,7 @@ setup(
                     "ddtrace.profiling.collector.stack",
                     sources=["ddtrace/profiling/collector/stack.pyx"],
                     language="c",
-                    extra_compile_args=["-DPy_BUILD_CORE"],
+                    extra_compile_args=extra_compile_args,
                 ),
                 Cython.Distutils.Extension(
                     "ddtrace.profiling.collector._traceback",

